rabbit_monitor.py

```diff
@@ -1,13 +1,12 @@
 #!/usr/bin/env python
 #   coding: UTF-8
+import asyncio
 from collections import namedtuple
 import logging
-import sys
-from time import sleep
+import os
 
-import requests
-from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.util.retry import Retry
+import aiohttp
+from aiohttp import web
 from structlog import wrap_logger
 
 import settings
```

Comment:
Could we do:
`import settings as env`

to differentiate between values from settings module (which come straight from os.env) and the settings data structure.